workflows/argo/resolve-output-url.yaml

```diff
@@ -16,29 +16,15 @@ spec:
         command: [bash]
         source: |
           set -e
-          remove_slashes () {
-              echo $(echo $1 | sed -e 's~/~-~')
+          fail_on_slash () {
```

Comment:
I tested in my shell that this logic works.

Reply:
Yah, I tested it as well.